templates/company_management/create_company_profile.html

```diff
@@ -0,0 +1,68 @@
+                                <div class="step-content active" id="stepContent1">
+                                    <div class="row">
+                                        <div class="col-sm-6">
+                                            <div class="form-group">
+                                                <label for=""> Name </label>
+                                                <input class="form-control" placeholder="Name" name='name' type="text">
+                                            </div>
+                                        </div>
+                                        <div class="col-sm-6">
+                                            <div class="form-group">
+                                                <label for=""> Address </label>
+                                                <input class="form-control" placeholder="Address" name='address'
+                                                       type="text">
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="row">
+                                        <div class="col-sm-6">
+                                            <div class="form-group">
+                                                <label for=""> Contact
+                                                </label>
+                                                <input class="form-control" placeholder="Contact" name='contact'
+                                                       type="text">
+                                            </div>
+                                        </div>
+                                        <div class="col-sm-6">
+                                            {# Addedd spare column #}
+                                        </div>
+                                    </div>
+                                    <div class="form-group">
+                                        <label for=""> Company Type </label>
+                                        <select class="form-control select2" name='company_type' multiple="true">
+                                            <option value='vendor' selected="true"> Vendor</option>
+                                            <option value='client'> Client</option>
+                                        </select>
+                                    </div>
+                                    <div class="form-buttons-w text-right">
+                                        <button class="btn btn-primary">Submit Form
+                                        </button>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </form>
+                </div>
+            </div>
+        </div>
+    </div>
+
+{% endblock %}
```

Comment:
add new empty lines at the end of all files to remove this ret circle

Reply:
done